src/GovUk.Education.ExploreEducationStatistics.Common/Services/Interfaces/ICacheService.cs

```diff
@@ -1,34 +1,19 @@
-﻿#nullable enable
+#nullable enable
 using System;
 using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Common.Cache.Interfaces;
-using GovUk.Education.ExploreEducationStatistics.Common.Model;
-using Microsoft.AspNetCore.Mvc;
 
 namespace GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces
 {
-    public interface ICacheService
+    public interface ICacheService<in TKey> where TKey : ICacheKey
```

Comment:
I think it's a great use of the interfaces to support other caching providers in the future, and in a way that we don't have redundant parameters like the blob container in the cache key 🙂